pkg/scheduler/tests/operation_test.go

```diff
@@ -552,12 +551,10 @@ partitions:
 
 	waitForAvailableNodeResource(t, ms.scheduler.GetClusterContext(), "[rm:123]default",
 		[]string{"node-2:1234"}, 50000000, 1000)
+	waitForUpdatePartitionResource(t, partitionInfo, common.Memory, 150000000, 1000)
```

Comment:
Could you add check for `vcore` also?

Reply:
I would prefer check all resources in one function, thank you!

Reply:
sorry I changed my mind, checking the resources separately is easier to debug and logging